docs/reference/configuration/keywords.rst

```diff
@@ -908,6 +908,22 @@ and/or history matching project.
         The various prior distributions available for the ``GEN_KW``
         keyword are described :ref:`here <prior_distributions>`.
 
+        It is also possible to skip the templating functionality of ``GEN_KW``, it also allows
+        for two positional arguments;
+
+        ::
+
+                GEN_KW  ID  priors.txt
+
+        where :code:`ID` is an arbitrary unique identifier,
+        and :code:`priors.txt` is a file containing a list of parametrized keywords
+        and a prior distribution for each.
+
+        This way ``GEN_KW`` will not create an output file for each realization, but instead
+        add the parameter values to a file located in runpath called: ``parameters.json``.
+        A file named ``parameters.txt`` is also create which contains the same information,
+        but it is recommended to use ``parameters.json``.
+
```

Comment:
We should move this functionality up to emphasis that this is the default behavior.